bigquery/src/test/java/org/apache/zeppelin/bigquery/BigQueryInterpreterTest.java

```diff
@@ -65,17 +51,16 @@ public String getOne() {
     public String getWrong()  {
       return wrongQuery;
     }
-
   }
 
   @SuppressWarnings("checkstyle:abbreviationaswordinname")
-  protected static Constants CONSTANTS = null;
+  protected static Constants constants = null;
```

Comment:
CONSTATNS -> constants , if this also violate checkstyle ?

Reply:
`CONSTANTS` violates Checkstyle, because it is not declared as `final`. I added a new commit which removes the `SuppressWarnings` annotation from this field. It isn't useful at this point.